<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWidget</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <widget class="QWidget" name="authorization">
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>150</y>
       <width>400</width>
       <height>300</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Авторизация</string>
     </property>
     <widget class="QPushButton" name="enterButton">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>240</y>
        <width>160</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Войти</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="showPassword">
      <property name="geometry">
       <rect>
        <x>145</x>
        <y>170</y>
        <width>200</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Показать пароль</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>55</x>
        <y>120</y>
        <width>80</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Пароль</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>55</x>
        <y>50</y>
        <width>80</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Логин</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="loginLine">
      <property name="geometry">
       <rect>
        <x>145</x>
        <y>50</y>
        <width>200</width>
        <height>40</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="passwordLine">
      <property name="geometry">
       <rect>
        <x>145</x>
        <y>120</y>
        <width>200</width>
        <height>40</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
